testing/vulnerable_app/models/thing.py

```diff
@@ -0,0 +1,8 @@
+from ..core.database import Base
+
+
+class Thing(Base):
+    """
+    This is separate from Widget since reusing it causes flakiness.
```

Comment:
Can you elaborate on the flakiness you experience with this?

Reply:
See above comment.